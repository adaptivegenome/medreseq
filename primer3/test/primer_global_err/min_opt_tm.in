SEQUENCE_ID=min_opt_tm
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_MAX_TM=65
PRIMER_OPT_TM=60
PRIMER_MIN_TM=62
=
