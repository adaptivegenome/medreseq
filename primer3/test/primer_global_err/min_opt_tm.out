SEQUENCE_ID=min_opt_tm
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_MAX_TM=65
PRIMER_OPT_TM=60
PRIMER_MIN_TM=62
PRIMER_ERROR=Optimum primer Tm lower than minimum or higher than maximum
=
